pkg/controller/ticdc_control.go

```diff
@@ -14,11 +14,17 @@
 package controller
 
 import (
+	"bytes"
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
```

Comment:
`io/ioutil ` is deprecated since go1.16.

Reply:
The project still use go 1.13 in go.mod.